lib/selinuxtest.pm

```diff
@@ -73,4 +73,16 @@ sub check_fcontext {
     }
 }
 
+# check SELinux security category of a file/dir
+sub check_category {
```

Comment:
May I know which case is using the **check_category** function, actually I did not find the case actually. Thanks.

Reply:
good catching, I used/pasted wrong function (copy and paste do harms sometime).

Reply:
Hi Ben,
I have revised the code and did rerun (see new test run link above).
All cases are pass.
Please help to check again.

Reply:
Sure